api/src/main/java/org/itsallcode/openfasttrace/api/core/LinkedSpecificationItem.java

```diff
@@ -147,15 +148,53 @@ public SpecificationItem getItem()
     /**
      * Add a link to another item with a status.
      *
-     * @param item
-     *            the item to be linked to
-     * @param status
-     *            the link status
+     * @param item   the item to be linked to
+     * @param status the link status
      */
     public void addLinkToItemWithStatus(final LinkedSpecificationItem item, final LinkStatus status)
     {
         this.links.computeIfAbsent(status, key -> new ArrayList<>());
         this.links.get(status).add(item);
+        switch (status)
+        {
+        case COVERED_SHALLOW:
+        {
```

Comment:
Superfluous use of block (brackets).

Reply:
Use IntelliJs compatibility plugin to get same formatting as Eclipse.

Reply:
DONE